### ISSUE TITLE
Getting unknown block type advanced_backup setting 

### ISSUE THREAD

**Issue by Reeta580** [hashicorp/terraform-provider-aws#17354]
I am using terraform v0.12.14  
I am getting unknown block type **advanced_backup_setting** while using it in my tf code. Can someone please help. I have verified it is using provider.aws version 3.26

resource "aws_backup_plan" "test_backup" {
  name = "test_backup_plan"

  rule {
    rule_name         = "test_backup_rule"
    target_vault_name = aws_backup_vault.test.name
    schedule          = "cron(0 16 * * ? *)"
  }

  advanced_backup_setting {
    backup_options = {
      WindowsVSS = "enabled"
    }
    resource_type = "EC2"
  }
}

**Comment by ewbankkit** [comment]
@Reeta580 Thanks for raising this issue.
Could you please double check that you are indeed using **v3.26.0** of the AWS Provider?
I have verified that there is not an issue with the latest provider version:

```console
$ terraform plan
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.


------------------------------------------------------------------------

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # aws_backup_plan.test_backup will be created
  + resource "aws_backup_plan" "test_backup" {
      + arn     = (known after apply)
      + id      = (known after apply)
      + name    = "test_backup_plan"
      + version = (known after apply)

      + advanced_backup_setting {
          + backup_options = {
              + "WindowsVSS" = "enabled"
            }
          + resource_type  = "EC2"
        }

      + rule {
          + completion_window = 180
          + rule_name         = "test_backup_rule"
          + schedule          = "cron(0 16 * * ? *)"
          + start_window      = 60
          + target_vault_name = "target_vault"
        }
    }

Plan: 1 to add, 0 to change, 0 to destroy.

-----------------------------------------------------------------------
```

Support for `advanced_backup_setting` was added via #15341 and released with [**v3.11.0**](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md#3110-october-15-2020) of the AWS Provider.

**Comment by Reeta580** [comment]
Thanks @ewbankkit for confirming. Issue has been fixed. I was calling a tf 0.11 module in my code, I am not seeing any error after I replaced it with another 0.12 module.
